pkg/apisix/cluster.go

```diff
@@ -63,6 +77,7 @@ type ClusterOptions struct {
 type cluster struct {
 	name         string
 	baseURL      string
+	baseURLHost  string
```

Comment:
If host is required, we can change the `baseURL` to `pathPrefix`, currently, `baseURLHost` is partial of `baseURL`.